### PR TITLE
DCS-2142 enable service unavailable page for all environments

### PR DESCRIPTION
**Pull request by GurnankCheema**
**view on across all environments**

Screenshot 2023-03-30 at 13 13 34

### CODE REVIEW

File: feature.env [hunk 1]
```diff
@@ -17,3 +17,4 @@ REDIS_PORT=6379
 REDIS_TLS_ENABLED=false
 
 API_CLIENT_ID=use-of-force-client
+SERVICE_IS_UNAVAILABLE=false
```
Comment (GurnankCheema) [inline feature.env:20]: need integ tests to pass

File: server/views/service-unavailable.njk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        text: "Digital Prison Services",
+        href: links.exitUrl      
+    }) }}
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row govuk-body">
+        <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">This service is currently unavailable</h1>
+        <p>You will be able to use Report use of force at 8:00am on Wednesday 5 April 2023.</p>
```
Comment (andrewrlee) [inline server/views/service-unavailable.njk:17]: until? rather than at?

File: server/views/service-unavailable.njk [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <p>If you were reporting a use of force, we have not saved your work. You will have to start again later.</p>
```
Comment (andrewrlee) [inline server/views/service-unavailable.njk:18]: something about "or writing statements"?

File: server/views/service-unavailable.njk (hunk 2 above)
Comment (GurnankCheema) [reply server/views/service-unavailable.njk:17]: do you mean
You will be able to use Report use of force until 6:00am on Wednesday 5 April 2023.
?

File: server/views/service-unavailable.njk (hunk 3 above)
Comment (GurnankCheema) [reply server/views/service-unavailable.njk:18]: like this?
If you were reporting a use of force or writing statements, we have not saved your work. You will have to start again later.

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**